<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(45, 101, 197);
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(50, 110, 255);
	color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Save Chages</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_widget">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>129</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>531</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Task</string>
   </property>
  </widget>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.542289, y1:0.057, x2:0.56204, y2:0.903, stop:0 rgba(91, 139, 223, 255), stop:1 rgba(6, 201, 229, 255));}</string>
   </property>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(45, 101, 197);
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(50, 110, 255);
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Delete Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_quote">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(45, 101, 197);
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(50, 110, 255);
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Get Quote</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="quoteTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addtask_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(45, 101, 197);
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(50, 110, 255);
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Add Task</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/assets/taskify_logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="urgent_task">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background-color: rgb(255, 142, 0);
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(255, 71, 0 );
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select to Highlight Urgent Task:</string>
    </property>
   </widget>
  </widget>
  <zorder>bgwidget</zorder>
  <zorder>save_button</zorder>
  <zorder>list_widget</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>taskLineEdit</zorder>
 </widget>
 <resources>
  <include location="taskify_logo_resource.qrc"/>
 </resources>
 <connections/>
</ui>
